Lib/fontmake/instantiator.py

```diff
@@ -147,6 +147,8 @@
     "woffMinorVersion",
 }
 
+DEFAULT_LAYER_NAME = "foreground"
```

Comment:
this is not standard, it might be what RoboFont uses maybe but it's not in the UFO spec.

Reply:
Yeah, it’s the RoboFont default. Removed.